### ISSUE TITLE
BLD: (DOC) some deeply nested lists look bad in PDF

### ISSUE THREAD

**Issue by jfbu** [scipy/scipy#8655]
Consider this in case of deep nesting:

![capture d ecran 2018-04-02 a 12 59 54](https://user-images.githubusercontent.com/2589111/38194103-cadcf6f8-3675-11e8-9540-496bf54fbaaf.png)

or also this

![capture d ecran 2018-04-02 a 12 56 37](https://user-images.githubusercontent.com/2589111/38193998-52a86046-3675-11e8-91f7-00652c48022c.png)

with short terms.

The size of the margin is set here in ``source/conf.py``.

```latex
\renewenvironment{description}%
{\begin{latexdescription}[\setleftmargin{60pt}\breaklabel\setlabelstyle{\bfseries\itshape}]}%
{\end{latexdescription}}
```

A LaTeX hack could possibly be added to test nesting level and use something much smaller for the ``Parameters, Returns, Raises`` etc... where anyway the nested description starts on next line.

### Scipy/Numpy/Python version information:
```
1.1.0.dev0+3a3eadf 1.14.2 sys.version_info(major=3, minor=6, micro=5, releaselevel='final', serial=0)
```

**Comment by jfbu** [comment]
Here is how it will look like with #8657:

![capture d ecran 2018-04-02 a 14 27 15](https://user-images.githubusercontent.com/2589111/38196367-03f10742-3683-11e8-8e2d-9f41c7060419.png)

(the commit number is same because the changes were not yet committed)

Parameters are allowed `50pt` width, not `60pt` and are bold, not bold italic.

At top level bold italic and only `15pt` indent.

**Comment by ilayn** [comment]
I think there will be no valid solutions as PDF is really not suitable for this type of documentation style. So as long as there is a magical solution we don't need to introduce hacks that would increase the tech debt on the maintenance.

**Comment by jfbu** [comment]
Using `60pt` indentation at each nesting level quickly gets confronted with the page size limits. Modifying `60pt` to `30pt` is not a hack and would quite improve the situation.

But, as pointed out [there](https://github.com/scipy/scipy/pull/8654#issuecomment-377918038), this would reveal still another bug of `expdlist` which is incompatibility with usages in a table.

Thus, to avoid all those problems, perhaps my best recommendation would be to drop altogether usage of `expdlist` LaTeX package which dates back to 1999 and has core bugs.

**Comment by ilayn** [comment]
Yes I agree it's quite suboptimal. 

However, given that a great majority of the userbase uses online docs, I am a bit skeptical about its worth. Obviously, having a better PDF is desired but I don't know if anybody has the time for a major overhaul or reviewing the overhaul afterwards.

**Comment by jfbu** [comment]
Dropping entirely usage of `expdlist` gives this screenshot (to be compared to the ones above).

![capture d ecran 2018-04-02 a 17 43 03](https://user-images.githubusercontent.com/2589111/38202833-6d01e02a-369d-11e8-9628-485e0a3f085d.png)

(the funny boldened list under Method available is caused by issue #8658).

Thus, this would basically solve the problem with nested lists.

My taste would be to use these small indentations but make sure parameter name is always on line of its own (i.e. always emulate the ``\breaklabel``  of `expdlist`). Sure, this would make document longer. But aslo less cluttered.

- one can obtain this now by a hack,
- or perhaps a LaTeX package exists providing the feature,
- or Sphinx LaTeX writer could add enhanced markup and it could be used for that effect.

**Comment by jfbu** [comment]
Fixed (essentially) at c881fdea7a7ef7c518b7898cfd53b2880af06778 (refs #8657)
